pkgs/by-name/gi/git-town/package.nix

```diff
@@ -13,7 +13,7 @@
 
 buildGoModule rec {
   pname = "git-town";
-  version = "21.0.0";
+  version = "21.1.0";
 
   src = fetchFromGitHub {
```

Comment:
The `version` gets propagated into the source definition automatically, but you will have to update the `hash` manually, otherwise the file will not get downloaded and the old one is reused instead.

Reply:
Thx, I was running my head against this for a few minutes 😆